instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -55,22 +57,34 @@
 from opentelemetry.trace import SpanKind, TracerProvider, get_tracer
 from opentelemetry.util._importlib_metadata import version as util_version
 
+if TYPE_CHECKING:
+
+    class wrapt:  # pylint: disable=invalid-name
+        class ObjectProxy:
+            def __init__(self, *args: Any, **kwargs: Any) -> None:
+                pass
+else:
+    import wrapt
```

Comment:
This messed a bit with sphinx... 😓

Reply:
I'm on the process of adding type hints to wrapt, so I've removed this logic here...

ref.: https://github.com/GrahamDumpleton/wrapt/pull/279

Reply:
Actually.. It was not this.. But I've reverted anyway, better to wait for wrapt..